virttest/qemu_vm.py

```diff
@@ -1252,6 +1252,15 @@ def set_value(opt_string, key, fallback=None):
 
             return secret_cmdline + " -spice %s" % (",".join(spice_opts))
 
+        def add_thread_context(devices, params):
+            for thread_context in params.objects("vm_thread_contexts"):
+                thread_context_params = params.object_params(thread_context)
+                dev = devices.thread_context_define_by_params(thread_context_params, thread_context)
+                set_cmdline_format_by_cfg(dev, self._get_cmdline_format_cfg(),
+                                          "vm_thread_contexts")
```

Comment:
I think renaming the type name to `thread_context(s)` would be better.
`_get_cmdline_format_cfg` is in the qemu command context, so the `vm_` prefix is unnecessary.

Also, the parameter names are not uniform. I think dev type defines the format of a kind of device, so it doesn't need to be represented in the plural(like thread_context**s**, nic**s**).

Reply:
Updated now, thanks!